src/pages/dashboard/_StateCumulativeTestsContainer.js

```diff
@@ -104,12 +144,20 @@ export default function CumulativeTestsByStateContainer() {
     }
   `)
 
-  const data = groupAndSortStateDaily(query)
+  const [useTestsPerCapita, setUseTestsPerCapita] = useState(false)
+
+  const toggleChartData = () => setUseTestsPerCapita(u => !u)
+
+  const allData = useMemo(() => {
+    return groupAndSortStateDaily(query)
+  })
```

Comment:
unless you pass an array of dependencies to `useMemo` it will perform the calculation on every render.

so here you want `useMemo(()=>groupAndSortStateDaily(query),[query])`

https://reactjs.org/docs/hooks-reference.html#usememo

Reply:
if only react was smart enough to just use the variables use from within the memoized function...

Reply:
@goleary check changes in 959f53a. I've memoized `allData` as suggested, and kept `useMemo` for the other two values (updating with `useTestsPerCapita`). Doing it this ensures that if other functionality is added, we're not resetting the `data` state for the charts unnecessarily. Thoughts?